src/FixedDataTableBufferedRows.js

```diff
@@ -50,7 +50,6 @@ var FixedDataTableBufferedRows = React.createClass({
   },
 
   componentDidMount() {
-    setTimeout(this._updateBuffer, 1000);
```

Comment:
This looks like my work. I probably deleted the logic and moved it to the reducer but forgot to clean this up.

this lgtm